src/channels/mutations.ts

```diff
@@ -0,0 +1,34 @@
+import makeMutation from "@saleor/hooks/makeMutation";
+import gql from "graphql-tag";
+
+import { ChannelCreate, ChannelCreateVariables } from "./types/ChannelCreate";
+
+export const channelErrorFragment = gql`
```

Comment:
Let's move fragments to `src/fragments/channels.ts` to be consistent with the latest changes made in https://github.com/mirumee/saleor-dashboard/pull/592